examples/js/loaders/ColladaLoader.js

```diff
@@ -969,11 +969,11 @@ THREE.ColladaLoader.prototype = {
 
 			if ( data.bindShapeMatrix ) {
 
-				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose(); 
+				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose();
 
 			} else {
 
-				build.bindMatrix = new THREE.Matrix4().transpose(); 
+				build.bindMatrix = new THREE.Matrix4().identity();
```

Comment:
neither transpose nor identity make sense on new matrix 😕

Reply:
Please read https://github.com/mrdoob/three.js/pull/14475#discussion_r202517423

It's their for clarity.